modules/workers/nodepools.tf

```diff
@@ -57,7 +57,7 @@ resource "oci_containerengine_node_pool" "workers" {
       for_each = var.cni_type == "npn" ? [1] : []
       content { # VCN-Native requires max pods/node, nsg ids, subnet ids
         cni_type          = "OCI_VCN_IP_NATIVE"
```

Comment:
Shouldn't you specify this for flannel as well?

Reply:
The attribute is applicable only for npn: https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/containerengine_node_pool#max_pods_per_node